admin/app/controllers/cache/ImageDecacheController.scala

```diff
@@ -40,10 +39,11 @@ class ImageDecacheController extends Controller with Logging with AuthLogging wi
 
       val originUri = new URI(originUrl)
 
-      ImageServices.clearFastly(originUri)
-      ImageServices.clearImgix(originUri)
+      val imageServices = new ImageServices(wsClient)
```

Comment:
I don't know if decache is called intensibely, if it is it may be nicer to put the object creation outside the method (as an attribute of the controller for instance)